_includes/item/child/compound-item-download-buttons.html

```diff
@@ -31,7 +31,11 @@
             {% unless child.object_location == nil %}
             <li>
                 <a target="_blank" rel="noopener" href="{{ child.object_location | relative_url }}" class="dropdown-item">
-                    {% if page.title != child.title and child.title != nil %}{{ child.title | truncatewords: 3 }}{% else %}Item {{ forloop.index }}{% endif %}
+                    {% if child.title != nil %}
+                        {{ child.title | truncatewords: 3 }}
```

Comment:
@koilebeit I am in favour of using the filename instead of title.

Reply:
Are you sure? Isn't the filename the omeka id like 414c9cc176d60658d6ba8f3fa4d96d7b9ee1490f.jpg?
I have also just noticed that the download does not work at all when I click on download:
![image](https://github.com/Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch/assets/73133626/0dbf3afc-13eb-4c3d-b9bc-5976c63223f6)

Reply:
> Are you sure? Isn't the filename the omeka id like 414c9cc176d60658d6ba8f3fa4d96d7b9ee1490f.jpg? I have also just noticed that the download does not work at all when I click on download: ![image](https://private-user-images.githubusercontent.com/73133626/340235396-0dbf3afc-13eb-4c3d-b9bc-5976c63223f6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MzI4NzgsIm5iZiI6MTcxODYzMjU3OCwicGF0aCI6Ii83MzEzMzYyNi8zNDAyMzUzOTYtMGRiZjNhZmMtMTNlYi00YzNkLWI5YmMtNTk3NmM2MzIyM2Y2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE3VDEzNTYxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUwNTlkMTQyMDM1MGJiNjVlNmNjM2QyY2QzNTMzMmYzMWRlYjJjOWUxY2NhYjVhM2VkMTJjZGNmNjdkZmMwNTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0zkVCIWX1WGvfAxkwae9-ibLVtUcwNQRK9z1kkKnMlw)

I opened a ticket at the DH Unibe. We should use the id.extension (i.e. abb161644.jpeg) as filename with this neat HTML trick

```html
<a href="path/to/your/file.txt" download="desired_filename.txt">Download File</a>
```

Reply:
Unfortunately, the trick does not work if the medium is not from the same origin url as is the case with us because the resources are at omeka:

> Firefox only allows users to download files of the same origin [due to a security concern](https://bugzilla.mozilla.org/show_bug.cgi?id=676619). The file must come from your own server or domain name, otherwise it will be opened in the browser. 
    While downloading cross-origin files is allowed in Chrome and the latest Opera (with Chromium/Blink), they will both ignore the attribute value. In other words, the file name will remain unchanged.

[https://webdesign.tutsplus.com/quick-tip-using-the-html5-download-attribute--cms-23880t](https://webdesign.tutsplus.com/quick-tip-using-the-html5-download-attribute--cms-23880t)

Reply:
Damn... Than I am out of ideas.
Ps: Sebi fixed the omeka server.